python/RobotControl/RunningWithSSH.py

```diff
@@ -25,7 +34,8 @@ def run_script_on_robot(id: int, script: str) -> str:
         Returns: 
             An error message or an empty string.
     """
-    robot.ssh.write_script(script)
+    augmented_script = augment_script(script)
```

Comment:
To be honest i dont know if this is the right place to do this. Maybe the augment responsibility should be in the function calling this.